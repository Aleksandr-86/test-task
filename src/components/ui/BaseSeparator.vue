<template>
  <div
    class="flex px-1 items-center before:mt-0.5 before:flex-1 before:border-t before:border-neutral-300 after:mt-0.5 after:flex-1 after:border-t after:border-neutral-300">
    <p class="mx-4 mb-0 text-center font-semibold">Или</p>
  </div>
</template>
