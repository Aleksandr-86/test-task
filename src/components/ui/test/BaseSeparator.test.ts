import { render } from '@testing-library/vue'
import BaseSeparator from '@/components/ui/BaseSeparator.vue'

describe('Компонент BaseSeparator', () => {
  it('структура соответствует снимку', () => {
    const wrapper = render(BaseSeparator, {})

    expect(wrapper).toMatchSnapshot()
  })
})
