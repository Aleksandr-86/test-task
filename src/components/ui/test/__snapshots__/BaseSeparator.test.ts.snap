// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Компонент BaseSeparator > структура соответствует снимку 1`] = `
{
  "baseElement": <body>
    <div>
      <div
        class="flex px-1 items-center before:mt-0.5 before:flex-1 before:border-t before:border-neutral-300 after:mt-0.5 after:flex-1 after:border-t after:border-neutral-300"
      >
        <p
          class="mx-4 mb-0 text-center font-semibold"
        >
          Или
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex px-1 items-center before:mt-0.5 before:flex-1 before:border-t before:border-neutral-300 after:mt-0.5 after:flex-1 after:border-t after:border-neutral-300"
    >
      <p
        class="mx-4 mb-0 text-center font-semibold"
      >
        Или
      </p>
    </div>
  </div>,
  "debug": [Function],
  "emitted": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "html": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
